src/Day01.kt

```diff
@@ -2,23 +2,25 @@
 fun main() {
     fun part1(input: List<String>): Int {
         var checkSum = 0
-        for (values in input) {
-            val num = values.split(' ')
-            checkSum += num.maxOf { it.toInt() } - num.minOf { it.toInt() }
+        for (line in input) {
+            val values = line.split("\t")
```

Comment:
Also, not sure how you made this work with splitting on spaces, unless you went to the trouble of modifying the input file. Instead you can use the `\t` escape character to split on tabs.